module/TaskManagement/src/TaskManagement/Service/NotificationService.php

```diff
@@ -49,9 +50,15 @@ public function sendTaskClosedInfoMail(Task $task);
 	/**
 	 * Send notification to members that a new Work Item Idea has been created
 	 * @param Task $task
-	 * @param User $member
+	 * @param User | NULL $member
 	 * @param OrganizationMembership[] $memberships
 	 * @return BasicUser[] receivers
 	 */
-	public function sendWorkItemIdeaCreatedMail(Task $task, User $member, $memberships);
+	public function sendWorkItemIdeaCreatedMail(Task $task, $member, $memberships);
+	/**
+	 * Send import details from Kanbanize to all members of an organization
+	 * @param array $result
+	 * @param Organization $organization
+	 */
+	public function sendKanbanizeImportResultMail($result, Organization $organization);
```

Comment:
Come detto sopra metti questo mail in un NotificationService nel modulo Kanbanize. Qui non ha senso